src/js/features/progress.js

```diff
@@ -312,7 +346,9 @@ Object.assign(MediaElementPlayer.prototype, {
 		media.addEventListener('timeupdate', (e) => {
 			if (media.duration !== Infinity ) {
 				player.setProgressRail(e);
-				player.setCurrentRail(e);
+				if(!t.forcedHandlePause){
```

Comment:
Is this conditional necessary? If it is, make sure you do the same in the `progress` listener